.github/workflows/PR.yml

```diff
@@ -18,14 +18,16 @@ jobs:
 
       steps:
          -  name: Checkout the repo
-            uses: actions/checkout@v2
+            uses: actions/checkout@v3
 
          -  name: Setup JDK
-            uses: actions/setup-java@v2
+            uses: actions/setup-java@v3
             with:
                distribution: 'zulu'
                java-version: '8'
 
+         -  uses: gradle/gradle-build-action@v2
+
```

Comment:
https://github.com/gradle/gradle-build-action

Ref https://github.com/detekt/detekt/pull/3723, it's really so fast to run jobs, and many other advantages.